design/apidsl/action.go

```diff
@@ -403,7 +403,7 @@ func Params(dsl func()) {
 //	})
 //
 func Payload(p interface{}, dsls ...func()) {
-	payload(false, p, dsls...)
+	payload(false, false, p, dsls...)
```

Comment:
The second argument to `payload`can be removed now I think...

Reply:
Sorry, fixed it. https://github.com/goadesign/goa/pull/1582/commits/f7d765af7f76651b72c190e2c094f017f2ad25af